packages/containers/src/HomeListView/HomeListView.component.js

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from 'react';
+import { api } from 'react-cmf';
+import { Layout } from 'react-talend-components';
+
+import AppHeaderBar from '../AppHeaderBar';
+import List from '../List';
+import SidePanel from '../SidePanel';
+
+function HomeListView(props) {
```

Comment:
a bit of destructuring here will help the user to discover the component interface